internal/apis/certmanager/validation/warnings.go

```diff
@@ -21,4 +21,6 @@ package validation
 const (
 	// deprecatedACMEEABKeyAlgorithmField is raised when the deprecated keyAlgorithm field for an ACME issuer's external account binding (EAB) is set.
 	deprecatedACMEEABKeyAlgorithmField = "ACME issuer spec field 'externalAccount.keyAlgorithm' is deprecated. The value of this field will be ignored."
+
+	deprecatedHMacMD5Value = "Usage of HMACMD5 is deprecated in cert-manager v1.8 and will no longer be supported in cert-manager v1.9. Please use one of  HMACSHA1, HMACSHA256, HMACSHA512 instead"
```

Comment:
nitpick: HMAC is an acronym (hash-based message authentication code) so I think usually the variable name would be written differently (although I recognise the library doesn't do this, e.g. `dns.HmacSHA512`)

```suggestion

	deprecatedHMACMD5Value = "Usage of HMACMD5 is deprecated in cert-manager v1.8 and will no longer be supported in cert-manager v1.9. Please use one of HMACSHA1, HMACSHA256, HMACSHA512 instead"
```

Reply:
Good catch, I initially put `Hmac` everywhere as that was in the library, then changed it, but this was left behind